kitsune/settings.py

```diff
@@ -806,13 +796,10 @@ def JINJA_CONFIG():
 
 #
 # Sessions
-SESSION_COOKIE_AGE = config('SESSION_COOKIE_AGE', default=4 * 7 * 24 * 60 * 60, cast=int)  # 4 weeks
```

Comment:
What is the reason to remove it? It is a django settings

Reply:
This has been restored. Probably this is not marked as outdated because I moved around a few commits in order to make them more atomic. Nice catch